pm/sender.go

```diff
@@ -86,14 +86,6 @@ func (s *sender) validateSender(info *SenderInfo) error {
 		return ErrSenderValidation{fmt.Errorf("unable to validate sender: deposit and reserve is set to unlock soon")}
```

Comment:
Shouldn't we skip this check if price is zero?

Reply:
Fixed in 1964464 - see [this comment](https://github.com/livepeer/go-livepeer/pull/1616#issuecomment-698581224) for more details on the overall changes.